docs/pages/versions/unversioned/sdk/updates.mdx

```diff
@@ -20,6 +20,8 @@ The expo-updates library allows you to programmatically control and respond to n
 
 Most of the methods and constants in this module can only be used or tested in release mode; they do not make sense in debug builds where you always load the latest JavaScript from your computer while in development.
 
+(It is possible to [build a debug version of your app where all these methods can be used](/eas-update/debug-updates/#debugging-of-native-code-in-an-app), but such an app will not open the latest JavaScript from your computer -- it will load published updates just as a release build would.)
```

Comment:
@amandeepmittal I added a note here in the Updates SDK docs, since it says "only release mode" can be used and that is no longer completely true.

Reply:
Thanks for adding this note!